common/azure-common/src/main/java/com/azure/common/exception/AzureLibraryException.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
```

Comment:
Copyright header is wrong style. Checkstyle should be complaining if you run it. If it isn't complaining, checkstyle must not be set up correctly

Reply:
Done. Haven't setup the checkstyle for Common yet. Fixed it by copying from other files.

Reply:
No - it is the wrong text entirely. CheckStyle enforces a different style for all track two client library code.

Reply:
It should be this:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
Updated